applications/serial_lte_modem/src/main.c

```diff
@@ -53,6 +53,16 @@ static void indicate_wk(struct k_work *work);
 
 BUILD_ASSERT(CONFIG_SLM_WAKEUP_PIN >= 0, "Wake up pin not configured");
 
+NRF_MODEM_LIB_ON_INIT(serial_lte_modem_init_hook, on_modem_lib_init, NULL);
+
+/* Initialized to value different than success (0) */
+static int modem_lib_init_result = -1;
```

Comment:
Global static should not initialized, will get build warning

Reply:
That's only for initializing to zero (because that's unnecessary), this should be fine.

Reply:
No more build warning (Zephyr SDK) so I was just overconcerned.